.github/workflows/playwright.yaml

```diff
@@ -0,0 +1,40 @@
+name: 'playwright-test-report'
+on:
+  pull_request:
+    workflow_dispatch:
+jobs:
+  test-report:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
+
+    - name: Install dependencies
+      run: npm ci
+
+    - name: Run Playwright tests
+      run: npm run test:coverage
+      env:
+        CI: true
+
+    - name: Code Coverage Summary Report
+      uses: irongut/CodeCoverageSummary@v1.3.0
+      with:
+        filename: 'coverage/cobertura-coverage.xml'
+        badge: true
+        format: 'markdown'
+        hide_complexity: true
+        thresholds: '65 85'
```

Comment:
What does threshold mean in this context?

Reply:
As you can see, in the comment there's health marker. Thresholds are used to define the value < 65 = x, 65-85 = -, >85 = v.
Also, they should influence the color of the badge for coverage. We do not fail the build due to low coverage, for now.